ria_remote/create_sibling_ria.py

```diff
@@ -0,0 +1,391 @@
+    dataset tree layout and a file of the same name per each dataset directory determining object tree layout version
+    (we already switch from dirhashlower to dirhashmixed for example) and an additional directory `error_logs` at the
+    toplevel.
+    """
+
+    # TODO: option to skip existing remotes in case of recursive?
+
+    _params_ = dict(
+        dataset=Parameter(
+            args=("-d", "--dataset"),
+            doc="""specify the dataset to process.  If
+            no dataset is given, an attempt is made to identify the dataset
+            based on the current working directory""",
+            constraints=EnsureDataset() | EnsureNone()),
+        url=Parameter(
+            args=("url",),
+            metavar="URL",
+            doc="""""",
+            constraints=EnsureStr() | EnsureNone()),
+        name=Parameter(
+            args=('-s', '--name',),
+            metavar='NAME',
+            doc="""sibling name to create for this publication target.
+                If `recursive` is set, the same name will be used to label all
+                the subdatasets' siblings. When creating a target dataset fails,
+                no sibling is added""",
+            constraints=EnsureStr() | EnsureNone()),
+        ria_remote=Parameter(
+            args=("--ria-remote",),
+            metavar="RIAREMOTE",
+            doc="""name of the RIA storage sibling (git-annex special remote). Must not be identical to NAME. 
+            By default NAME is appended with '-ria'""",
+            constraints=EnsureStr() | EnsureNone()),
+        force=Parameter(
+             args=("-f", "--force"),
+            doc="""don't fail on existing siblings. Use and possibly reconfigure them instead.""",
+            action='store_true'),
+        post_update_hook=Parameter(
+            args=("--post-update-hook",),
+            doc="""Enable git's default post-update-hook on the remote end""",
+            action="store_true"),
+        shared=Parameter(
+            args=("--shared",),
+            metavar='{false|true|umask|group|all|world|everybody|0xxx}',
+            doc="""if given, configures the access permissions on the server
+        for multi-users (this could include access by a webserver!).
+        Possible values for this option are identical to those of
+        `git init --shared` and are described in its documentation.""",
+            constraints=EnsureStr() | EnsureBool() | EnsureNone()),
+        group=Parameter(
+            args=("--group",),
+            metavar="GROUP",
+            doc="""Filesystem group for the repository. Specifying the group is
+        particularly important when [CMD: --shared=group CMD][PY:
+        shared="group" PY]""",
+            constraints=EnsureStr() | EnsureNone()),
+        no_ria_remote=Parameter(
+            args=("--no-ria-remote",),
+            doc="""don't establish a ria-remote in addition to the sibling NAME.""",
+            action="store_true"),
```

Comment:
Might be better to have a positive parameter: `ria_remote=True` and that cmdline option, but `dest='ria_remote` and `store=False`.

Reply:
Done now.